packages/matrix-client-server/src/utils/setupTokens.ts

```diff
@@ -121,6 +124,38 @@ export async function setupTokens(
       validated_at: epoch(),
       added_at: epoch()
     })
+
+    await clientServer.matrixDb.insert('threepid_validation_session', {
+      session_id: 'validatedSession',
+      medium: 'msisdn',
+      address: '0612938719',
+      client_secret: 'validatedSecret',
+      last_send_attempt: 1,
+      validated_at: epoch()
+    })
+
+    await clientServer.matrixDb.insert('user_threepids', {
+      user_id: '@validated:example.com',
+      medium: 'msisdn',
+      address: '0612938719',
+      validated_at: epoch(),
+      added_at: epoch()
+    })
+
+    await clientServer.matrixDb.insert('access_tokens', {
+      id: randomString(64),
```

Comment:
WHat is the need of this "id" field ? Since not collected

Reply:
It's the primary key of the 'access_tokens' table in the synapse DB schema so I needed to fill it even if I'm not collecting it